config/defaults.yml

```diff
@@ -1,8 +1,8 @@
-aws_access_key_id:
-aws_secret_access_key:
+aws_access_key_id: "{{ lookup('env','AWS_ACCESS_KEY_ID') }}"
```

Comment:
+1. All of the changes in this defaults file I like. Please just add information about this in the Readme so that people know where the ssh key is read from along with which env vars must be set.

Reply:
Done